tests/cross_functional/kcs/test_monitor_recovery.py

```diff
@@ -53,7 +53,7 @@
 @magenta_squad
 @system_test
 @ignore_leftovers
-@pytest.mark.order("last")
+@pytest.mark.order("last-1")
```

Comment:
What about:
@pytest.mark.order("second_to_last")
https://pytest-order.readthedocs.io/en/stable/usage.html#order-relative-to-other-tests